src/raftstore/store/store.rs

```diff
@@ -535,13 +535,19 @@ impl<T: Transport, C: PdClient> Store<T, C> {
         loop {
             match self.apply_res_receiver.as_ref().unwrap().try_recv() {
                 Ok(ApplyTaskRes::Applys(multi_res)) => {
+                    let mut raftlog_gc_tasks = RaftlogGcTasks::new();
```

Comment:
can we use` len(multi_res)` as the initialized capacity? In most cases, the ready res has only CompactLog command.